src/Pages/Dashboard/dashboardStyle.jsx

```diff
@@ -64,4 +73,49 @@ const uploadStyle = theme => ({
   }
 });
 
-export { dashboardStyle, monitoringStyle, uploadStyle };
+const historyStyle = theme => ({
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+    backgroundColor: primaryColor
+  },
+  select: {
+    "&:before": {
+      borderColor: "white",
+    },
+    "&:after": {
+      borderColor: "white",
+    },
+    color: "white",
+  },
+  inputLabel: {
+    color: "white"
+  },
+  formControl: {
+    color: "white"
+  },
+  icon: {
+    fill: "white"
+  },
+  input: {
+    color: "white"
+  }
+});
+
+const materialTheme = createMuiTheme({
+  palette: {
+    primary: {
+      light: secondaryColor,
+      main: primaryColor,
+      dark: backgroundColor,
+      contrastText: secondaryColor,
+    },
+    error: {
+      main: warningColor,
+    },
+  }
+});
```

Comment:
Was this the only workaround to having non-clashing styles on the DatePicker? I'm not a particularly big fan of having multiple theme/theme providers in our code, since it splinters the style. 

Eventually, I would like to refactor the code such that our styling is consistent enough that we don't need all the `*style.jsx` files. 

I already removed a whole bunch of ThemeProviders in the old code that didn't do anything since it was already applied at a higher level in the root, so I would prefer not doing this unless we really had to.

Reply:
I moved the theme to `mainStyle.jsx`, is that what you want? I see there is also mainTheme in mainStyle.jsx as well but Ithnk something else is using it so I won't touch it.

Reply:
Not necessarily, I'm just not a fan of having multiple themes. I say leave it as is for now, and we'll fix it when we overhaul the existing theme (since it's all white anyway so it's pretty bad). 

If you look at `src/index.js`, you can see we've got a top level theme provider and I want to keep it that way moving forward.

Reply:
I took another look into the theme situation. It seems that many `*style.jsx` files have many specific consigurations only for the respective page (e.g. closing animation of the sidebar) that squashing them into one universal theme might not be practical. I guess it could still work using unique class names, but I feel like it might cause further confusion when maintaining.

My suggestion would be to define universal theme elements (like colors, font, etc.) in `mainStyle.jsx` and stick to them (instead of using `#fff`, for example) so that changing style will be easier.

We could also move all themes into `mainStyle.jsx` so that it will be one file, but contains all separate themes.

What do you think?